src/definition/accessors/ISchedulerExtend.ts

```diff
@@ -0,0 +1,8 @@
+import { IJob } from '../scheduler';
+import { IJobBuilder } from './IJobBuilder';
+
+export interface ISchedulerExtend {
```

Comment:
I think this interface is missing some features.

You want to be able to unschedule jobs and get a list of scheduled jobs

Reply:
So, what I'm going to do is add a new accessor to to the modify stage to allow modification to the jobs. But if I get into it and that is not a fluid design or works in reality, then we will see how else we can work with this.